### PR TITLE
Notification templates

### PR DESCRIPTION
**Pull request by ssatagopan**
# [Notification Templates for Hosting System ](https://github.com/OHS-Hosting-Infrastructure/infrastructure/issues/162)

A set of notification templates to be used when sending messages during incidents.

Changes proposed in this pull request:

- Initial version of templates

### CODE REVIEW

File: docs/how-we-work/notification-templates/acknowledgment-message.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+The message should be sent within 2 hours of receipt during [core business hours](https://github.com/OHS-Hosting-Infrastructure/infrastructure/tree/main/docs/how-we-work#working-hours)
+If message was received outside code business hours, it should be sent next business day.
+
+If an issue was reported as a ticket, please use ticket to acknowledge receipt using mentions in a comment.
+
+If an issue was reported as an email, send an email response and ask for a ticket.
+
+## When to Use
+If an issue is P1 or P2 according to docs/how-we-work/support-process.md
```
Comment (kilbergr) [inline docs/how-we-work/notification-templates/acknowledgment-message.md:11]:
```suggestion
If an issue is P1 or P2 according to [our support process doc definition](../support-process.md)
```

File: docs/how-we-work/notification-templates/initial-message.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+The following should be used as a message template when sending notifications about system outages and this is the first email letting people know.
+
+## When to Use
+If an issue is determined to be P0 or P1 based on docs/how-we-work/support-process.md
```
Comment (kilbergr) [inline docs/how-we-work/notification-templates/initial-message.md:4]:
```suggestion
If an issue is determined to be P0 or P1 based on [our Support Process doc definition](support-process.md).
```

File: docs/how-we-work/notification-templates/resolved-message.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+The following should be used as a message template when sending notifications about system outages and this is when the issue has been resolved.
```
Comment (kilbergr) [inline docs/how-we-work/notification-templates/resolved-message.md:1]: Mention this is via email?

File: docs/how-we-work/notification-templates/update-message.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+The following should be used as a message template when sending notifications about system outages and this is an update about an issue that is still happening.
```
Comment (kilbergr) [inline docs/how-we-work/notification-templates/update-message.md:1]: Do you think it's worth mentioning this is via email?

**Review by kilbergr — CHANGES_REQUESTED** [review]

File: docs/how-we-work/notification-templates/resolved-message.md (hunk 3 above)
Comment (ssatagopan) [reply docs/how-we-work/notification-templates/resolved-message.md]: Added

File: docs/how-we-work/notification-templates/update-message.md (hunk 4 above)
Comment (ssatagopan) [reply docs/how-we-work/notification-templates/update-message.md]: Added

**Review by kilbergr — APPROVED** [review]
LGMT TY

**Pull request merged**